src/_App.scss

```diff
@@ -17,12 +18,26 @@
   }
 }
 
+// TODO: Make this class generalizable
+.home {
+  padding-bottom: 50px;
+  &:nth-child(odd) {
```

Comment:
I wonder what happen if not the 'odd' children have the color background, but instead the 'even' children like this design: https://www.figma.com/file/KiY0iNeez6ZkqdqP35SLS6/Homepage-Modules?node-id=1519%3A322. We might also have cases in the future when the background color only apply to some even or odd children. What do you think?

Reply:
Also if we later add additional components on the home page, the odd/even order might get messed up.